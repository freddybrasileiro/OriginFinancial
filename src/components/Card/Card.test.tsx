import * as React from 'react';
import { shallow } from 'enzyme';
import { Card } from './index';

describe('Card', () => {
  it('Should render the Card component', () => {
    expect(shallow(<Card>Content here</Card>)).toMatchSnapshot();
  });
});
