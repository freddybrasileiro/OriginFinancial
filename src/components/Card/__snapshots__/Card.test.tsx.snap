// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Should render the Card component 1`] = `ShallowWrapper {}`;
